packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/v1/__init__.py

```diff
@@ -39,6 +47,57 @@ def _instrument(self, **kwargs):
             "Completions.create",
             completion_wrapper(tracer),
         )
+
+        chat_token_counter = meter.create_counter(
```

Comment:
No, it doesn't throw exceptions when metering is disabled, but leads to unnecessary metrics records; adding checks before all meter.create_xxx to avoid this.